README.md

```diff
@@ -171,6 +171,35 @@ If you plan to only work on the API you only have to build the server architectu
 * Point your browser to:
     * [http://localhost:5000/api-docs](http://localhost:5000/api-docs)
 
+## Docker setup and testing for (non)developers
```

Comment:
I suggest to move this into a separate file. Probably it makes sense to have one for the whole docker setup. I expect there to be more to be documented.

Reply:
Check!. I agree. We would need to include commands such as database migration, coverage among others